velox/exec/Task.cpp

```diff
@@ -502,6 +505,47 @@ void Task::initTaskPool() {
       fmt::format("task.{}", taskId_.c_str()), createTaskReclaimer());
 }
 
+void Task::initDriverFactory() {
+  VELOX_CHECK_NULL(
+      consumerSupplier_,
+      "Serial execution mode doesn't support delivering results to a "
+      "callback");
+
+  taskStats_.executionStartTimeMs = getCurrentTimeMs();
```

Comment:
The task stats executionStartTimeMs moves the Task constructor, not sure for that.

Reply:
This code has similar purpose with https://github.com/facebookincubator/velox/pull/13075?

Reply:
Yes, if https://github.com/facebookincubator/velox/pull/13075 can merge, it's ok.